script.js

```diff
@@ -0,0 +1,27 @@
+const textsToCopyContainer = document.querySelector('#textsToCopyContainer')
+const paragraphs = []
+
+fetch('https://flipsum-ipsum.net/api/icw/v1/generate?ipsum=recipe-ipsum-text-generator&start_with_fixed=0&paragraphs=4').then((response) => {
+    return response.json()
+}).then((data) => {
+    data.forEach((textOutput) => {
+        paragraphs.push(textOutput.split(' '))
```

Comment:
although super efficient, this is quite hard to read - it may be easier to do an extra line with more semantic code, that shows first the array of words being made, and then this array being added to the paragraphs array. It's very clever! But took us a while to understand!

Reply:
done!